armi/physics/executers.py

```diff
@@ -163,9 +172,25 @@ class DefaultExecuter(Executer):
     but is not required. The sequence look like:
 
     .. impl:: Default tool for executing external calculations.
-        :id: I_ARMI_EX2
+        :id: I_ARMI_EX1
         :implements: R_ARMI_EX
 
+        Facilitates the execution of external calculations by accepting ``options`` (an
+        :py:class:`~armi.physics.executers.ExecutionOptions` object) and providing
+        methods that build run directories and execute a code based on the values in
+        ``options``.
+
+        The :py:meth:`~armi.physics.executers.DefaultExecuter.run` method will first
+        resolve any derived options in the ``options`` object and check if the specified
+        ``executablePath`` option is valid, raising an error is not. If it is,
+        preparation work for executing the code is performed, such as performing any geometry
+        transformations specified in subclasses or building the directories needed
+        to save input and output files. Once the temporary working directory is created,
+        the executer moves into it and runs the external code, applying any results
+        from the run as specified in subclasses.
+
+        Finally, any geometry perturbations that were performed are undone.
+
     * Choose modeling options (either from the global run settings input or dictated programmatically)
```

Comment:
Looks like this .. impl:: section is located between a colon and the list introduced by it. Not sure how this will be rendered but even just for readers of the source code, it would be better to relocate the impl section such that the list can be right after "The sequence look like:".

Reply:
Yeah, I think the `impl` section should be moved down below that list.

Reply:
https://github.com/terrapower/armi/pull/1594/commits/16d81f36c13f1453b5bc5acf8c55f7af5a529903